Dockerfile

```diff
@@ -1,16 +1,15 @@
-FROM python:3.7.3-alpine3.9
+FROM python:3
```

Comment:
Thanks for the callout and looking over this, @tym-xqo! I definitely ran into some issues with 3.10, so I'll revert to 3.9 for now. I got some toolchain improvements with ubuntu in other environments (despite how heavy the download is), but debian might be a good middle ground for image size vs speed of build.

Reply:
Haha I just happened to see this PR when it went in, because the repo is public, and I guess I was still watching it, as well as following you 🙂 

I seriously use debian-slim for all base images now when available and do recommend it!